### DISCUSSION TITLE
最新版本汉化错误、打开文件错误。

### DISCUSSION THREAD

**Discussion by leonardo-zi** [Moonvy/figmaEX#42] [General]
设备: Macbook Pro M3
设备系统版本: macOS Sequoia 15.1
FigmaEX版本: 2024/08/16 - v124.3.1 客户端版本

问题：
1.界面汉化错误，图1
2.文件内画板命名错误，字体和图1相同，但是在figma官网打开，画板命名没有被修改 （最新版本进不去文件没有截图）
3.文件打不开，图2
![iShot_2024-08-19_11 40 31](https://github.com/user-attachments/assets/0f05beba-ba97-4f01-ae62-6def402de4c1)
![iShot_2024-08-19_11 40 47](https://github.com/user-attachments/assets/c4631b9f-a0e5-4324-b95a-6241e0fc988f)

**Comment by NoahShipwright** [comment]
【文件打不开】遇到一样的报错，解决了吗？

**Comment by yArna** [comment]
这个问题是因为 macOS 15 应用名有“中文”会导致的

升级到最新版 FigmaEX 客户端就可以了
